inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/MessageEventReconciliationUtil.kt

```diff
@@ -259,17 +259,16 @@ internal interface MessageEventReconciliationUtil {
         @SuppressWarnings("FunctionMaxLength")
         private fun getNumTimesToSatisfyTriggersForDisplay(message: Message): Int {
             val maxImpression = message.getMaxImpressions()
-            val displayedImpression: Int = LocalDisplayedMessageRepository.instance().numberOfTimesDisplayed(message)
-            val incrementRemoved = message.getCampaignId()?.let {
```

Comment:
Remove all related code for this function.

Reply:
Done